### DISCUSSION TITLE
Discord Rich Presence: "Playing <X>" line could show "Playing Game Name (on Stadia)" instead of "Playing Stadia" + note

### DISCUSSION THREAD

**Discussion by ivucica** [ChristopherKlay/StadiaEnhanced#221] [Ideas & Requests]
Hi,

I don't quite mind the current implementation showing "Playing Stadia", but it's also less useful: person wanting to see what exactly I play has to look at the details of my account.

In its place, it may make sense to offer an option to use different strings: for instance, "Playing Destiny 2 (on Stadia)" or even "Playing Destiny 2" with the mention of _Stadia_ in the note (rather than mentioning the game in the note).

**Comment by ChristopherKlay** [comment]
I would love to offer this and in fact would likely use it as the default - but i (afaik) can't.

The "Playing Stadia" is due to the registered Discord presence being labeled "Stadia". It's not possible to change the actual presence name for each individual game, unless you manually register/manage a entire presence for each title.
